app/backend/templates/digest.md

```diff
@@ -0,0 +1,19 @@
+---
+subject: "You missed {{ len(notifications)|couchers_escape }} notifications on Couchers.org"
+---
+
+{% from "macros.html" import button, link, support_email, email_link, newline %}
+
+Hi {{ notification.user.name|couchers_escape }}!
+
+This is your Couchers.org digest of notifications you missed.
+
+{% for notification in notifications %}
+
+Title: {{ notification.title|couchers_escape }}
+
+Content: {{ notification.content|couchers_escape }}
+
+Debug: (topic, action) = ({{ notification.topic|couchers_escape }}, {{ notification.action|couchers_escape }})
```

Comment:
What's the plan for this?

Reply:
This debug thing can go, but we need to redo the emails at some point.